### PR TITLE
Spark stoploss improvements [sc-12044]

### PR DESCRIPTION
**Pull request by piotrwitek**
## Changes 👷‍♀️
  <Please add short list of changes>

- Fix slider for opening aave positions [sc-12044]
- Fix issues with app config for aave like [sc-12042]
  
## How to test 🧪
  <Please explain how to test your changes>

- test slider when opening aave / spark positions and when adjusting

### CODE REVIEW

File: components/GenericMultiselect.tsx [hunk 1]
```diff
@@ -162,7 +162,7 @@ export function GenericMultiselect({
   const outsideRef = useOutsideElementClickHandler(() => setIsOpen(false))
   const scrollRef = useRef<HTMLDivElement>(null)
   const optionsFeatureFlagsArray = options.map((option) =>
-    option.featureFlag ? getAppConfig('features')[option.featureFlag] : true,
+    option.featureFlag ? getLocalAppConfig('features')[option.featureFlag] : true,
```
Comment (marcinciarka) [inline components/GenericMultiselect.tsx:165]: Thats within the context, shouldn't it be `useAppConfig` now?

File: features/automation/protection/stopLoss/sidebars/SidebarAdjustStopLossEditingStage.tsx [hunk 2]
```diff
@@ -197,7 +197,7 @@ export function SidebarAdjustStopLossEditingStage({
             }
             sliderPercentageFill={sliderPercentageFill}
             lastValue={stopLossState.stopLossLevel}
-            maxBoundry={sliderMax.times(2)}
+            maxBoundry={sliderMax}
```
Comment (marcinciarka) [inline features/automation/protection/stopLoss/sidebars/SidebarAdjustStopLossEditingStage.tsx:200]: Isn't this needed by some other token pairs?

File: helpers/config/access-config-context.ts [hunk 3]
```diff
@@ -14,16 +14,16 @@ import { configLSOverridesKey } from './constants'
...
+export function useAppConfig<T extends ConfigResponseTypeKey>(configKey: T): ConfigResponseType[T] {
   try {
-    let ac = accessContext(configContext)
+    let ac = useContext(configContext)
     if (!ac) {
       throw new Error("ConfigContext not available! getAppConfig can't be used serverside")
     }
     if (window.localStorage) {
-      ac = merge(ac, {
+      ac = merge<ConfigContext, ConfigContext>(ac, {
```
Comment (marcinciarka) [inline helpers/config/access-config-context.ts:24]: ❤️

File: components/GenericMultiselect.tsx (hunk 1 above)
Comment (piotrwitek) [reply components/GenericMultiselect.tsx:165]: yeah correct

**Review by marcinciarka — APPROVED** [review]

**Pull request merged**